tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -165,9 +165,9 @@ local utils = import 'templates/utils.libsonnet';
     accelerator: tpus.v3_8,
   },
   configs: [
-    dlrm + v3_8 + one_core + timeouts.Hours(3),
-    dlrm + v3_8 + seq_fwd + timeouts.Hours(3),
-    dlrm + v3_8 + mp_fwd + timeouts.Hours(3),
+    // dlrm + v3_8 + one_core + timeouts.Hours(3),
```

Comment:
If you think it's worth keeping this config around, can you take them off the schedule with [`mixins.Experimental`](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/4397c3976074fe29ec8c506a4e30343f9d40fb55/templates/mixins.libsonnet#L33)? This will still leave them available to run manually on an ad-hoc basis

If you don't think we need these configs anymore, please remove these lines and the corresponding mixins altogether.

Reply:
I see, let's go with `mixins.Experimental` for future reference.

Reply:
Thanks @will-cromar , so I assume that the `functional::schedule` will be overwritten by `::Experimental::schedule` ?

Reply:
Yeah, that's right